examples/hpc/dla-driver/src/lib.rs

```diff
@@ -804,6 +820,7 @@ impl Dla {
             self.handshake_disable_hw();
             return false;
         }
+        sprintln!("Finishing handshake");
```

Comment:
Consider if prints in driver code are necessary or if they should be put behind a feature flag :)

Reply:
Print removed